### PR TITLE
Dev to Master, v0.2.3

### PR DESCRIPTION
**Pull request by benmalka**
- Support -P option
- Hardkill in Windows (In case the process isn't being closed)
- Handle long post data
- Event-based console API
- Enable cookie extraction
- Fix Documentation
- Fix bugs in the new features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-Foxdriver [![Build Status](https://travis-ci.com/saucelabs/foxdriver.svg?token=px5tFzamGvYgujeyYVEp&branch=master)](https://travis-ci.com/saucelabs/foxdriver)
```
Comment (christian-bromann) [inline README.md:1]: It would be good if we could keep the badge

**Review by christian-bromann — CHANGES_REQUESTED** [review]
@benmalka these changes look great but it seems that you have pushed them to your own fork. Would you mind to clean up the PR (revert changes to e.g. the package name `@benmalka/foxdriver`) and have these merged upstream?

**Comment by benmalka** [comment]
As you can see there are tons of changes since I've forked this project. How would you like to address all of them?

**Comment by christian-bromann** [comment]
> How would you like to address all of them?

@benmalka is there a way we can address conflicts and get this merged?

**Comment by benmalka** [comment]
@christian-bromann I've opened a new PR #22

**Pull request closed without merging**